Viper/Viper/App/SceneDelegate.swift

```diff
@@ -13,8 +14,13 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     ) {
         guard let windowScene = (scene as? UIWindowScene) else { return }
         window = UIWindow(windowScene: windowScene)
-        let viewController = ViewController()
-        window?.rootViewController = viewController
+        let builder = AssemblyBuilder()
```

Comment:
```suggestion
        let assemblyBuilder = AssemblyBuilder()
```
я бы называла экземпляр также как и класс